### PR TITLE
Fgts 81 enemy takes damage from player tongue

### PR DESCRIPTION
**Pull request by jDenyes**
added in functionality for tongue to damage "ant" enemies. This functionality will be built upon for collectable objects like the star and the flies / other enemies

### CODE REVIEW

**Comment by fheysel** [comment]
We need some kind of sound/visual indication that the enemy has taken damage and an animation for its death so that it doesnt just disappear. This might have to wait until the next demo.

**Comment by jDenyes** [comment]
I'd rather include this functionality for now without death animations than not include it all. 
I build up the code for tongueable stars from this functionality. If we really don't want this in we can include the code changes but disable the collision itself with the tongue collision layer

**Comment by fheysel** [comment]
the stars arent tongueable when I play on this branch

**Comment by jDenyes** [comment]
@fheysel Stars shouldn't be tongueable on this branch.

File: godotproject/PlayerTongue.gd [hunk 1]
```diff
@@ -7,13 +7,18 @@ signal tongue_swing
 export (float) var shoot_speed = 1500 * 0.6
 export (float) var max_shoot_dist = 400 * 0.6
 
+const COLLISION_LAYER_TONGUEABLE 		= 0x3
+const COLLISION_LAYER_TONGUE_CAN_DAMAGE = 0x6
+
```
Comment (charasyn) [inline godotproject/PlayerTongue.gd]:
```suggestion
const COLLISION_LAYER_TONGUEABLE        = 0x3
const COLLISION_LAYER_TONGUE_CAN_DAMAGE = 0x6

```

File: godotproject/PlayerTongue.gd [hunk 2]
```diff
@@ -7,13 +7,18 @@ signal tongue_swing
...
 var idle = true
 var shooting = false
 var swinging = false
 
 var shoot_direction = Vector2.RIGHT
 var tongue_length = 0
 
+var tongue_damage = 50
+
```
Comment (charasyn) [inline godotproject/PlayerTongue.gd]:
```suggestion
var tongue_damage = 1

```

File: godotproject/enemy-Ant.gd [hunk 3]
```diff
@@ -17,11 +17,14 @@ var direction = -1
 var attackDamage = 1
 var attackState = STATE.idle_e
 var is_dead = false
+
+var tongue_can_damage = true
+var health = 100
```
Comment (charasyn) [inline godotproject/enemy-Ant.gd]:
```suggestion
var health = 2
```

**Review by charasyn — CHANGES_REQUESTED** [review]
Here are a few style changes:
- We should have tongue damage and health be 1 and 2, instead of 50 and 100
- Align text using spaces instead of tabs

**Review by charasyn — APPROVED** [review]

**Pull request merged**